omod/src/main/resources/config.xml

```diff
@@ -142,4 +153,31 @@
 		<description>Identifier type of patient to generate AdhocTeleconsultation link eg: 'Patient Identifier' or 'National ID' etc</description>
 	</globalProperty>
 
+	<globalProperty>
+		<property>sms.enableAppointmentBookingSMSAlert</property>
+		<defaultValue>false</defaultValue>
+	</globalProperty>
+
+	<globalProperty>
+		<property>sms.enableAppointmentReminderSMSAlert</property>
+		<defaultValue>false</defaultValue>
+	</globalProperty>
+
+	<globalProperty>

```

Comment:
We should not be setting property like this. This attribute should be associated with location. Also where is this used? if not used, please remove.